Create SUPPORT.md (#508)
dotnet/Silk.NET: .github/SUPPORT.md

# Getting Support

> For support, please either use [GitHub Discussions](https://github.com/dotnet/Silk.NET/discussions) or contact as via [Discord](https://discord.com/invite/DTHHXRt)

Reviewer: ```suggestion
For support, please either use [GitHub Discussions](https://github.com/dotnet/Silk.NET/discussions) or chat to us on [Discord](https://discord.com/invite/DTHHXRt).
```

> <a href="https://discord.com/invite/DTHHXRt"><img src="https://discordapp.com/api/guilds/521092042781229087/widget.png?style=banner2" alt="Banner image of the Silk.NET Discord server"/></a>

Reviewer: Can we defer Discord's official badges, as these brought a lot of "Sex Dating" bots into the OpenTK server when we used them over there.

Author: What do you mean?

Reviewer: As in we had a lot of bots join the server using the link that posted links entitled "Sex Dating"

Author: Hmm, I mean we advertise this everywhere anyways, so why not here as well?

Reviewer: The discord link is fine, but it's something about the official widget.png in particular that attracts them.

Reviewer: ```suggestion
<a href="https://discord.com/invite/DTHHXRt"><img src="https://img.shields.io/badge/chat%20on-discord-7289DA" alt="Banner image of the Silk.NET Discord server"/></a>
```
